tutoraspects/templates/aspects/build/aspects-superset/localization/locale.yaml

```diff
@@ -785,8 +785,8 @@ fr_CA:
   Students: "\xC9tudiants"
   Superset: Superset
   Superset Active Users: Utilisateurs actifs du surensemble
-  Superset Active Users Over Time: Superset des utilisateurs actifs au fil du temps
-  Superset Registered Users Over Time: "Superset des utilisateurs enregistr\xE9s au fil du temps"
+  Superset Active Users Over Time: Surensemble des utilisateurs actifs au fil du temps
```

Comment:
@bmtcril does `Superset` can be translated?

Reply:
No, it shouldn't be. I wonder if these are machine translations...